gst/gst_object.go

```diff
@@ -97,3 +97,22 @@ func (o *Object) Unref() {
 func (o *Object) AddControlBinding(binding *ControlBinding) {
 	C.gst_object_add_control_binding(o.Instance(), binding.Instance())
 }
+
+// TODO: Consider wrapping GstObject GST_OBJECT_LOCK/GST_OBJECT_UNLOCK functionality
+// due to following flags related functionality is based on a regular uint32 field
+// and is not considered thread safe
```

Comment:
okay, issue for this: https://github.com/go-gst/go-gst/issues/132

Reply:
I think It is not an urgent issue, but it needs some discussion from different people who use those flags. In my case - I only set them in construction phase where multithreading is not an issue yet.